src/utils.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+function make_db(): PDO
+{
+    $db = new PDO('mysql:host=db; dbname=car-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function create_list_of_cars(array $cars): void
```

Comment:
This function should return the string rather than echoing it - always best practice to avoid echoing inside functions. 

Once that's done, this function should then be unit tested